src/Dev/Deprecation.php

```diff
@@ -234,14 +234,22 @@ public static function get_enabled()
 
     public static function isEnabled(): bool
     {
+        // Return early if disabled
         if (Environment::hasEnv('SS_DEPRECATION_ENABLED')) {
-            $envVar = Environment::getEnv('SS_DEPRECATION_ENABLED');
-            return self::varAsBoolean($envVar);
+            if (!Environment::getEnv('SS_DEPRECATION_ENABLED')) {
```

Comment:
I'd say don't nest this one, just do 

```php
if (Environment::hasEnv('SS_DEPRECATION_ENABLED') && !Environment::getEnv('SS_DEPRECATION_ENABLED')) {
```

Reply:
Pending decision on https://github.com/silverstripe/silverstripe-framework/pull/11055#discussion_r1398521409